pkg/event/target/scram_client.go

```diff
@@ -0,0 +1,36 @@
+package target
+
+import (
+	"crypto/sha256"
+	"crypto/sha512"
+	"hash"
+
+	"github.com/xdg/scram"
+)
+
+var SHA256 scram.HashGeneratorFcn = func() hash.Hash { return sha256.New() }
```

Comment:
Rename all the variables with kafka prefix to avoid namespace collisions in future.

Reply:
Updated